### ISSUE TITLE
About the VoVnet-99 checkpoint and config

### ISSUE THREAD

**Issue by zeyuwang615** [Megvii-BaseDetection/BEVDepth#118]
Hi, thanks for the great work!

I wonder if you could release the VoVnet-99 version checkpoint and config, so that the test set results can be reproduced?

Also, I wonder what pre-trained checkpoint for VoVnet-99 did you use? Was it first pre-trained on the COCO dataset for 2D Detection, and then pre-trained on the DDAM-15M dataset for depth prediction? Did you freeze any layer/stage in VoVnet-99 when training for 3D Detection?

Thanks a lot!

**Comment by swtju14** [comment]
Hi, @zeyuwang615 . We adopt DD3D pretrained VoVNet (V2-99) as backbone network for images. No freeze when training.

**Comment by zeyuwang615** [comment]
Does 'DD3D pretrained VoVNet (V2-99)' means the backbone is pre-trained sequentially on COCO and DDAM-15M as described in the paper?

![捕获](https://user-images.githubusercontent.com/26880977/207989233-e1031292-06a0-45d3-92d1-46fc92f86b9c.JPG)

More importantly, is there any checkpoint for such DD3D pretrained VoVNet (V2-99) available? I have managed to find two, but not sure which is the right one (or neither is): here in dd3d codebase
![tempsnip](https://user-images.githubusercontent.com/26880977/207989825-64c575f7-d683-4229-a61a-0093ba26ac74.png)

or dded_det_final.pth [here](https://drive.google.com/drive/folders/1h5bDg7Oh9hKvkFL-dRhu5-ahrEp2lRNN) in detr3d codebase.

Thanks a lot!

**Comment by swtju14** [comment]
dded_det_final.pth [here](https://drive.google.com/drive/folders/1h5bDg7Oh9hKvkFL-dRhu5-ahrEp2lRNN) in detr3d codebase.

**Comment by thom966** [comment]
Could you please provide the vovnet.py file that you used when loading pre-trained weights?

**Comment by swtju14** [comment]
We use vovnet.py file like [this](https://github.com/megvii-research/PETR/blob/30d32dadccdde9a524d57842967d08125cfafaa4/projects/mmdet3d_plugin/models/backbones/vovnet.py)
